lib/manager/components/transform/CustomCSVForm.js

```diff
@@ -34,9 +35,11 @@ const CustomCSVForm = (props: Props) => {
   } = props
 
   useEffect(() => {
+  // Default to true
+    const validateHeaders = props.validateHeaders !== undefined ? props.validateHeaders : true
```

Comment:
Do ES Lint rules apply to this repo? I know there are rules for a multiline ternary.

Reply:
> Do ES Lint rules apply to this repo? I know there are rules for a multiline ternary.

Datatools has its own eslint config